docs/optimizing-with-codeflash/benchmarking.mdx

```diff
@@ -65,13 +65,13 @@ Codeflash will then calculate the impact (if any) of any optimization on the per
    ```
    
 
-4. **Run Codeflash in CI**
+4. **Run Codeflash :**
 
-   Benchmark mode is best used together with Codeflash as a Github Action. This way, with every PR, you will know the impact of Codeflash's optimizations on your benchmarks.
+   Benchmark mode is best used together with Codeflash as a GitHub Action. This way, if new code in a Pull Request touches the benchmark workflow, Codeflash will try to optimize the code for the benchmark, and you will know the impact of Codeflash's optimizations on your benchmarks.
```

Comment:
nit: ```Codeflash will try to optimize the code for the benchmark```

Maybe we can write something like "Codeflash will trace through your benchmarks step to optimize the functions modified in your PR which touches the benchmark workflow, and you will ..